test/mongodb.test.js

```diff
@@ -1032,6 +1032,27 @@ describe('mongodb connector', function() {
   });
 
   describe('updateAll', function() {
+    it('should not delete the id from the update object', async function() {
+      let user = await User.create({name: 'Al', age: 31, email: 'al@strongloop'});
+      const userId = user.id;
+      user = user.toObject();
+      user.age = 100;
+
+      await User.update(user);
+      user.should.have.property('id', userId);
+    });
+
+    it('should reflect the changes in the model object', async function() {
```

Comment:
The test name is misleading. IIUC, it is testing `update() does not mutate the input model object`.